mimer-config/src/main/java/com/avanza/astrix/config/PropertyParser.java

```diff
@@ -57,4 +64,21 @@ public Integer parse(String value) {
 			return Integer.parseInt(value);
 		}
 	}
+
+	class StringListParser implements PropertyParser<List<String>> {
+		@Override
+		public List<String> parse(String value) {
+			return value.isBlank() ? Collections.emptyList() : List.of(value.split(","));
+		}
+	}
+
+	class IntListParser implements PropertyParser<List<Integer>> {
+		@Override
+		public List<Integer> parse(String value) {
+			return value.isBlank() ? Collections.emptyList() : Arrays.stream(value.split(","))
+					.map(String::trim)
+					.map(Integer::parseInt)
```

Comment:
Very minor detail: should use valueOf to avoid unnecessary boxing
```suggestion
					.map(Integer::valueOf)
```